src/storage/logs.js

```diff
@@ -45,10 +46,18 @@ function storeRequest(request) {
   if (modifiedBody.password) {
     delete modifiedBody.password;
   }
+  let url = '';
+  try {
+    // some request urls can't be normalized and
+    // throw an error
```

Comment:
Which ones? Is this a potential issue for compareUrl?

Reply:
requests to the medmorph backend itself come in as relative, and the normalize function fails on "urls" like `/index`.  We don't use compareUrl anywhere this would be an issue, though.